tsdb/index/index.go

```diff
@@ -49,6 +49,8 @@ const (
 	FormatV2 = 2
 
 	indexFilename = "index"
+
+	ByteAlign = 16
```

Comment:
Do we need to expose it?

It would be 
```suggestion
	seriesByteAlign = 16
```

Reply:
fixed